encrypted-media-respec.html

```diff
@@ -3025,6 +3016,53 @@
       <h3>Implementation Requirements</h3>
       <p>This section defines implementation requirements - for both user agents and <a def-id="keysystems"></a>, including the <a def-id="cdm"></a> and server - that may not be explicitly addressed in the algorithms.</p>
 
+	    <section id="cdm-constraint-requirements">
+        <h3>CDM Constraints</h3>
+        <p>
+          User agent implementers MUST ensure that CDMs do not access any information, storage or system capabilities that are not reasonably required for playback of protected media using the features of this specification. 
+          Specifically, the CDM SHALL NOT:
+        </p>
+        <ul>
+          <li>
+            <p>
+              Access network resources, either local or remote, except via the user agent, or as part of the user agent, as explicitly permitted by this specification.
```

Comment:
What is the purpose of the "or as part of the user agent" fragment? The CDM is always logically distinct from the CDM regardless of how they are implemented, so I'm not sure what this adds. To me, it increases complexity and decreases clarity. I recommend we drop it.